src/data/license/en.json

```diff
@@ -42,4 +42,4 @@
       "text": "For all possible disputes between the licenser and the licensee, the courts at the licenser's domicile are responsible. Swiss law is exclusively applicable."
```

Comment:
Hier immer noch: "Swiss law"?? Ist das korrekt in mapset.io

Reply:
Solche Inhalte werden nochmal später überarbeitet, ist aber nicht Inhalt dieser PR

Reply:
OK